src/components/icons/AudioMicrophoneMutedIcon.scss

```diff
@@ -1,3 +1,7 @@
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-audio-microphone-muted-icon {
   path { fill: var(--wxc-icon--muted-color); }
+
+  :disabled & {
```

Comment:
this should be a generic rule for all icons, we could use the common `.wxc-icon` class